package.json

```diff
@@ -13,7 +13,9 @@
     "url": "https://github.com/DevExpress/DevExtreme.AspNet.Data.git"
   },
   "main": "js/dx.aspnet.data.js",
+  "types": "js/dx.aspnet.data.d.ts",
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
I think that this is a normal practice. The problem is in `jquery` package. We shouldn't fix it.
Also, there are users that use `typings` package instead of `@types/*`. So we could break compilation on some projects.

Note that `jquery` is listed in peerDependencies of `devextreme` package. So, if a user uses typescript it is required to install typings for packages that don't provide built-in types.